### PR TITLE
Add Flatiron institute multipole libraries to the package index

### PR DESCRIPTION
**Pull request by milancurcic**
Thanks @Beliavsky for finding them.

* [FMM3D](https://github.com/flatironinstitute/FMM3D)
* [fmm2d](https://github.com/flatironinstitute/fmm2d)

### CODE REVIEW

**Review by certik — APPROVED** [review]
Very cool, thanks for finding them.

File: _data/package_index.yml [hunk 1]
```diff
@@ -1231,6 +1231,18 @@
   tags: fluid-dynamics fluid-simulation computational-fluid-dynamics turbulence high-performance-computing hpc cfd
   version: none
 
+- name: FMM3D
+  github: flatironinstitute/FMM3D
+  description: Compute N-body interactions governed by the Laplace and Helmholtz equations, to a specified precision, in three dimensions, on a multi-core shared-memory machine.
+  categories: scientific
```
Comment (awvwgk) [inline _data/package_index.yml:1237]:
```suggestion
  categories: numerical
  tags: fast-multipole-method
```

File: _data/package_index.yml [hunk 2]
```diff
@@ -1231,6 +1231,18 @@
...
+  license: Apache-2.0
+
+- name: fmm2d
+  github: flatironinstitute/fmm2d
+  description: Compute N-body interactions governed by the Laplace and Helmholtz equations, to a specified precision, in two dimensions, on a multi-core shared-memory machine.
+  categories: scientific
```
Comment (awvwgk) [inline _data/package_index.yml:1243]:
```suggestion
  categories: numerical
  tags: fast-multipole-method
```

**Review by awvwgk — COMMENTED** [review]
I think those fit better under numerical than scientific as those are libraries which can be used to build scientific applications, rather than standalone scientific projects.

**Pull request merged**